File: src/main/java/no/nav/melosys/eessi/models/sed/nav/Arbeidsgiver.java

package no.nav.melosys.eessi.models.sed.nav;

import java.util.List;
import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonInclude.Include;
import lombok.Data;


@JsonInclude(Include.NON_NULL)
@Data
public class Arbeidsgiver {

    private Adresse adresse;

    private List<Identifikator> identifikator;

    private String navn;
}

File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/models/sed/nav/Arbeidsgiver.kt
package no.nav.melosys.eessi.models.sed.nav

import com.fasterxml.jackson.annotation.JsonInclude

@JsonInclude(JsonInclude.Include.NON_NULL)
data class Arbeidsgiver(
    var adresse: Adresse? = null,
    var identifikator: List<Identifikator>? = null,
    var navn: String? = null
)
